### DISCUSSION TITLE
What to be expected from the "prodMode" config option?

### DISCUSSION THREAD

**Discussion by cguilhermef** [ngneat/transloco#397] [Q&A]
Hi guys

I didn't saw any explanation about what is mean the "prodMode" configuration option is. Somebody can explain?

**Comment by shaharkazaz** [comment]
Hey @cguilhermef,
When `prodMode` is enabled Transloco will disable all console warnings.
I updated [the docs](https://ngneat.github.io/transloco/docs/config-options#prodmode) with this explanation 🙂
